misc/src/main/java/com/example/snippets/ActivityEmbeddingJavaSnippets.java

```diff
@@ -336,6 +319,23 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             super.onCreate(savedInstanceState);
             setContentView(R.layout.activity_main);
 
+            // [START android_activity_embedding_splitAttributesBuilder_java]
+            SplitAttributes.Builder splitAttributesBuilder = new SplitAttributes.Builder()
+                .setSplitType(SplitAttributes.SplitType.ratio(0.33f))
+                .setLayoutDirection(SplitAttributes.LayoutDirection.LEFT_TO_RIGHT);
+
+            if (WindowSdkExtensions.getInstance().getExtensionVersion() >= 6) {
```

Comment:
Whats the version number mean here? You might want to include a constant that describes this.

Reply:
The version number indicates the required build version. I took this snippet from a codelab. I can't find any docs that explain how the version numbers map to build versions. The closest I've come is https://developer.android.com/guide/sdk-extensions#extension-names. I filed b/406244784 to hopefully get this documented.